src/navbar/navbar.tsx

```diff
@@ -292,7 +291,7 @@ const Component = <T,>(
     const renderNavbar = () => {
         return (
             <Layout.Content stretch={isStretch}>
-                <Nav $compress={compress}>
+                <Nav $compress={compress} aria-label="Main navigation menu">
```

Comment:
suggestion: expose a `headerLabel` and `drawerLabel` prop to support customisation of the aria-label